models/labels/gnosis_safe/labels_gnosis_safe_ethereum.sql

```diff
@@ -0,0 +1,12 @@
+{{config(alias='gnosis_safe_ethereum')}}
```

Comment:
i believe the team has rebranded to just 'safe' now -- maybe we should reflect that in all folder/directories, file names, aliases, hardcoded columns, etc.

Reply:
Oh good to know I wasn't sure, done ✔